wave_matching/include/wave/matching/PointcloudXYZGD.hpp

```diff
@@ -0,0 +1,42 @@
+
+/************************************************************************
+ *
+ *
+ *  Copyright 2015  Arun Das (University of Waterloo)
```

Comment:
Change to `@authors` command?

Reply:
nah, they did write it. I just doubt that they actually retain copyright.

Reply:
I mean you can replace this header with one consistent with all our other files - Doxygen formatting, `@file` command, etc. Then in the same place, there can be `@authors` listed. The names and university and email addresses can appear there.